src/dev.omaremara.bugtracker/dev/omaremara/bugtracker/model/Report.java

```diff
@@ -0,0 +1,36 @@
+package dev.omaremara.bugtracker.model;
+
+import dev.omaremara.bugtracker.model.Assigne;
+import dev.omaremara.bugtracker.model.Project;
+import dev.omaremara.bugtracker.model.ReportLevel;
+import dev.omaremara.bugtracker.model.ReportPriority;
+import dev.omaremara.bugtracker.model.ReportType;
+
+public class Report {
+  public String uiName;
+  public String title;
+  public String description;
+  public ReportLevel level;
+  public ReportPriority priority;
+  public ReportType type;
+  public Project project;
+  public User assigne;
+  // function return all dispcription
+  class Report
+  (String uiName, String title, String description, ReportLevel level,
+   ReportPriority priority, ReportType type, Project project, User assigne) {
+    this.title = title;
+    this.assigne = assigne;
+    this.description = description;
+    this.level = level;
+    this.priority = priority;
+    this.project = project;
+    this.uiName = uiName;
```

Comment:
There is no `uiName` here. There should be an integer `id` attribute.

Reply:
why i want to insert uiName to database to make foriegn key to user name in table user

Reply:
A report doesn't have a "name". The assignee is the attribute that should be a foriegn key to the users table. But a name can't be used as primary key in the users table, so we will either use an id or an email.